### EXERCISE 1. Answer the following questions by looking at the distribution table or coding with Python.

In [2]:
import numpy as np
import pandas as pd
from scipy import stats
import seaborn as sns
import matplotlib.pyplot as plt

### 1- A salesperson has found that the probability of a sale on a single contact is approximately .3. If the salesperson contacts 10 prospects, what is the approximate probability of making at least one sale?

In [3]:
n = 10    # the number of contact
p = 0.3   # probability of a sale on a single contact
x = 0   # sale: this is P(X≥1)=P(1)+P(2)+P(3)+...+P(10)

1 - stats.binom.cdf(x, n, p)

0.9717524751

### 2- Ten coins are tossed simultaneously. Find the probability of getting
### (i) at least seven heads



In [4]:
n = 10    # the number of toss
p = 0.5   # probability of a head
x = 6   # at least seven heads P(X≥7)=P(7)+P(8)+P(9)+P(10)

1 - stats.binom.cdf(x, n, p)

0.171875

### (ii) exactly seven heads

In [5]:
stats.binom.pmf(k=7, n=10, p=0.5)

0.11718749999999999

### (iii)at most seven heads

In [6]:
x=7
stats.binom.cdf(x, n, p)

0.9453125

### EXERCISE 2. The cycle time for trucks hauling concrete to a highway construction site is uniformly distributed over the interval 50 to 70 minutes. What is the probability that the cycle time exceeds 65 minutes  if it is known that the cycle time exceeds 55 minutes?

#### 50 < x < 70, 
#### P(X>65| X>55) = P(X>65) / P(X>55) 
#### = (1/4)/(3/4) 
#### = 1/3
#### 0.3333

### EXERCISE 3. The width of bolts of fabric is normally distributed with mean 950 mm (millimeters) and standard deviation 10 mm.

### 1- What is the probability that a randomly chosen bolt has a width of between 947 and 958mm?


In [7]:
mu=950
sigma=10

stats.norm.cdf(x=958, loc=950, scale=10) - stats.norm.cdf(x=947, loc=950, scale=10)

0.406056023605556

### 2- What is the appropriate value for C such that a randomly chosen bolt has a width less than C with probability .8531?

In [8]:
stats.norm.ppf(q=0.8531, loc=950, scale=10)

960.4982190962642

Manuel solution: For P = 0.8531 z value is 1.05. When I multiply it by std i get 1.05*10 = 10.5 
Then,  I sum up it with mu value which is 950
Answer = 960.5 mm

### EXERCISE 4. Suppose scores on exams in statistics are normally distributed with an unknown population mean and a population standard deviation of 3 points. A random sample of 36 scores is taken and gives a sample mean (sample mean score) of 68. Find a confidence interval estimate for the population mean exam score (the mean score on all exams). Find a 90% confidence interval for the true (population) mean of statistics exam scores.

In [9]:
n = 36
mu = 68
s = 3 / np.sqrt(36)

z = stats.t.ppf((1+0.90)/2, n-1)

h = s * z

CI = (mu-h, mu+h)
CI

(67.15521377301812, 68.84478622698188)

In [11]:
#alternative way

lower_cutoff = stats.t.ppf(0.050, n - 1, loc = mu, scale = s)  
upper_cutoff = stats.t.ppf(0.950, n - 1, loc = mu, scale = s) 
lower_cutoff, upper_cutoff

(67.15521377301812, 68.84478622698188)

### EXERCISE 5.  What is the normal body temperature for healthy humans? A random sample of 130 healthy human body temperatures provided by Allen Shoemaker7 yielded 98.25 degrees and standard deviation 0.73 degrees. Give a 99% confidence interval for the average body temperature of healthy people.

In [17]:
n = 130
mu = 98.25
s = 0.73 / np.sqrt(130)

z = stats.t.ppf((1+0.99)/2, n-1)

h = s * z

CI = (mu-h, mu+h)
CI

(98.08260738705933, 98.41739261294067)